features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -367,6 +367,7 @@
   "authentication_policy.check_account_exist": true,
   "authentication.jit_provisioning.username_provisioning_url": "/accountrecoveryendpoint/register.do",
   "authentication.jit_provisioning.password_provisioning_url": "/accountrecoveryendpoint/signup.do",
+  "authentication.jit_provisioning.enable_enhanced_feature": true,
```

Comment:
Please create an issue to track the migration aspect.

Reply:
The issue[1] was opened to track the migration aspects.

[1] https://github.com/wso2/product-is/issues/12017